src/software/backend/unix_simulator_backend.cpp

```diff
@@ -51,6 +51,10 @@ UnixSimulatorBackend::UnixSimulatorBackend(
     robot_crash_listener.reset(new ThreadedProtoUnixListener<TbotsProto::RobotCrash>(
         runtime_dir + ROBOT_CRASH_PATH, [](TbotsProto::RobotCrash& v) {}, proto_logger));
 
+    robot_statistic_listener.reset(new ThreadedProtoUnixListener<RobotStatistic>(
+        runtime_dir + ROBOT_STATISTIC_PATH, [](RobotStatistic& v) {},
+        proto_logger));
```

Comment:
Nima: @Andrewyx in the future we should update `RobotStatistic` proto to be within the `TbotsProto` namespace. Though there is a chance the would mess with replay.